src/main/java/it/pagopa/transactions/services/TransactionsService.java

```diff
@@ -569,13 +569,13 @@ private Mono<NewTransactionResponseDto> projectActivatedEvent(
                                 .authToken(sessionDataDto.getSessionToken())
                                 .status(TransactionStatusDto.fromValue(transaction.getStatus().toString()))
                                 // .feeTotal()//TODO da dove prendere le fees?
-                                .origin(
-                                        Optional.ofNullable(transaction.getOriginType())
+                                .clientId(
+                                        Optional.ofNullable(transaction.getClientId())
                                                 .map(
-                                                        origin -> NewTransactionResponseDto.OriginEnum
-                                                                .fromValue(origin.toString())
+                                                        clientId -> NewTransactionResponseDto.ClientIdEnum
+                                                                .fromValue(clientId.toString())
```

Comment:
See other

Reply:
Fixed in https://github.com/pagopa/pagopa-ecommerce-transactions-service/pull/114/commits/32acff6de0f07d4b345e1381aaa4f15071993ff3